cishouseholds/edit.py

```diff
@@ -1,10 +1,27 @@
 from itertools import chain
+from typing import List
 from typing import Mapping
 
 import pyspark.sql.functions as F
 from pyspark.sql import DataFrame
 
 
+def convert_null_if_not_in_list(df: DataFrame, column_name: str, options_list: List[str]) -> DataFrame:
```

Comment:
I like the idea of allowing the list of options acceptable. 
Should we leave it as by default options "male" and "female"?